x-pack/test/functional/apps/security/trial_license/index.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { FtrProviderContext } from '../../../ftr_provider_context';
+
+export default function({ loadTestFile }: FtrProviderContext) {
+  describe('security app - login selector - trial license', function() {
```

Comment:
nit: the description for "login selector" should be covered by `./login_selector`'s `describe` block
```suggestion
  describe('security app - trial license', function() {
```

Reply:
Yep, thanks :+1: